website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -16,6 +16,14 @@ upgrade flow.
 
 ## Consul 1.11.0
 
+### 1.10 Compatibility <EnterpriseAlert inline />
+Consul Enterprise versions 1.10.0 through 1.10.4 contain a latent bug that
+causes those agents to deregister the Serf health check of the node when they 
```

Comment:
```suggestion
causes Consul client agents to deregister the Serf health check of the node when they 
```
Is this suggestion correct, Matt?

Reply:
Now that you mention it I _think_ it should just be client agents but am going to verify that with a test.

Reply:
@karl-cardenas-coding Does this sound better to you?